sdk/appcenter-distribute/src/main/res/values/appcenter_distribute.xml

```diff
@@ -14,7 +13,12 @@
     <string name="appcenter_distribute_dialog_actioned_on_disabled_toast">Distribute was disabled</string>
     <string name="appcenter_distribute_unknown_sources_dialog_message">For security, your device is set to block installation of apps obtained from unknown sources.</string>
     <string name="appcenter_distribute_unknown_sources_dialog_settings">Settings</string>
+    <!-- TODO: Add translations for the alias. -->
+    <string name="appcenter_distribute_downloading_update" translatable="false">@string/appcenter_distribute_downloading_mandatory_update</string>
```

Comment:
It is translatable, that would give the wrong message to customers.
Use `tools:ignore="MissingTranslation"` instead (also fixes the error).